packages/navigation-gizmo/src/SphereScript.ts

```
@@ -1,245 +1,38 @@
+    Entity,
+    Script
 } from "@galacean/engine";
 
-/** @internal */
 export class SphereScript extends Script {
-  private static _startQuat: Quaternion = new Quaternion();
-  private static _startPointer: Vector2 = new Vector2();
-  private static _startPos: Vector3 = new Vector3();
-  private static _startAxis: Vector3 = new Vector3();
-  private static _tempMat: Matrix = new Matrix();
+    private _xEntity: Entity;
```

ESLint

[error] Delete `··`

(prettier/prettier)